packaging/osx/buildpackage.sh

```diff
@@ -83,9 +88,10 @@ modify_plist "{JOIN_SERVER_URL_SCHEME}" "openra-${MOD_ID}-${TAG}" "${PACKAGING_O
 echo "Packaging zip archive"
 
 zip "${PACKAGING_INSTALLER_NAME}-${TAG}" -r -9 "${PACKAGING_OSX_APP_NAME}" --quiet --symlinks
-mv "${PACKAGING_INSTALLER_NAME}-${TAG}.zip" ${OUTPUTDIR}
 
 popd > /dev/null
 
+mv "${BUILTDIR}/${PACKAGING_INSTALLER_NAME}-${TAG}.zip" ${OUTPUTDIR}
```

Comment:
Why did you move this line but not do something similar with the windows script?